app/services/licences/fetch-return-versions.service.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+/**
+ * Fetches return requirements data needed for the view '/licences/{id}/set-up` page
+ * @module FetchReturnVersionsService
+ */
+
+const ReturnVersionModel = require('../../models/return-version.model.js')
+
+/**
+ * Fetches return requirements data needed for the view '/licences/{id}/set-up` page
+ *
+ * @param {string} licenceId - The licence id for the licence to fetch return requirements
+ *
+ * @returns {Promise<Object>} the data needed to populate the view licence page's set up tab
+ */
+async function go (licenceId) {
+  return _fetch(licenceId)
+}
+
+async function _fetch (licenceId) {
+  return ReturnVersionModel.query()
+    .where('licenceId', licenceId)
+    .select([
+      'id',
+      'startDate',
+      'endDate',
+      'status',
+      'reason'
+    ])
+    .orderBy([
+      { column: 'startDate', order: 'desc' }
+    ])
```

Comment:
NIT - If we can we try and chain the Objection.js calls as if we were writing the SQL itself

```suggestion
  return ReturnVersionModel.query()
    .select([
      'id',
      'startDate',
      'endDate',
      'status',
      'reason'
    ])
    .where('licenceId', licenceId)
    .orderBy([
      { column: 'startDate', order: 'desc' }
    ])
```